Cargo.toml

```diff
@@ -7,4 +7,7 @@ resolver = "2"
 rust-version = "1.65"
 
 [profile.release]
-strip = true
+strip = true      # Automatically strip symbols from the binary.
+lto = true        # Link-time optimization.
+opt-level = "s"   # Optimize for speed.
```

Comment:
`s` stands for "optimise for size" and is typically slightly slower than the default `3` and marginally slower than `2`.
In rate scenarios when `2` is faster than `3` (optimisation shenanigans), `s` will be faster than `3` but still slower than `2`.

Reply:
I just measured this, for me the default opt level (i. e. 3) is ~10% faster for `tree-sitter generate` for all grammars that I tested with.

Reply:
can you try out w/ 2?

we can make the fastest the default profile, and have one for size

Reply:
Thanks - I think 2 is a good balance then of compile time and speed